src/Microsoft.Health.Dicom.Core/Models/Operations/DicomOperation.cs

```diff
@@ -40,5 +40,10 @@ public enum DicomOperation
     /// <summary>
     /// Specifies an update operation that updates the Dicom attributes.
     /// </summary>
-    Update
+    Update,
+
+    /// <summary>
+    /// Specifies an operation to delete extended query tag.
+    /// </summary>
+    DeleteExtendedQueryTag
```

Comment:
That's a good point. Personally I think making them wait for a status right now is OK, since we are kind of faking a synchronous API anyways. Its expected to not be able to get the status if you don't wait for the response to complete. 

However Will showed me that we do have a property in `GetExtendedQueryTagEntry` where we return operation data for reindexing - https://github.com/microsoft/dicom-server/blob/89bebf34d5ff92577a2707ce222e6031e19b89fa/src/Microsoft.Health.Dicom.Client/Models/GetExtendedQueryTagEntry.cs#L52
If we extend that table to hold delete operation as well, we could return the operation details here so there is some way for a user to understand what is acting on their query tag. If we did that, I would leave `DeleteExtendQueryTag` here so they could query for the operation